filecache.go

```diff
@@ -186,15 +186,18 @@ func (c *FileCache) MaybeDownload(filename string) error {
 //
 func (c *FileCache) GetFileName(filename string) string {
 	hashedFilename := md5.Sum([]byte(filename))
-	hashedDir := fnv.New32().Sum([]byte(filename))
+	fnvHasher := fnv.New32()
+	// Write never returns a non-nil error
+	_, _ = fnvHasher.Write([]byte(filename))
```

Comment:
Seems pretty unlikely, but we should catch and at least log an error here, right?